### ISSUE TITLE
error: package directory 'python3_redirect' does not exist

### ISSUE THREAD

**Issue by mr-c** [google/python-subprocess32#44]
When testing https://github.com/google/python-subprocess32/blob/master/setup.py#L20 promised ability to install under Python3.x

```
$ pip install subprocess32==3.5.0rc1
Collecting subprocess32==3.5.0rc1
  Downloading https://files.pythonhosted.org/packages/28/91/d1283618eba07c4e8e18c58b3fd8b5ff3a8992fb652a3720535ddf2f2916/subprocess32-3.5.0rc1.tar.gz (54kB)
    100% |████████████████████████████████| 61kB 843kB/s 
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/subprocess32.egg-info
    writing top-level names to pip-egg-info/subprocess32.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/subprocess32.egg-info/dependency_links.txt
    writing pip-egg-info/subprocess32.egg-info/PKG-INFO
    writing manifest file 'pip-egg-info/subprocess32.egg-info/SOURCES.txt'
    /home/michael/cwltool/env3/lib/python3.5/site-packages/setuptools/version.py:1: UserWarning: Module _posixsubprocess was already imported from None, but /tmp/pip-install-gk2ke0g4/subprocess32 is being added to sys.path
      import pkg_resources
    subprocess32 == subprocess on Python 3.
    error: package directory 'python3_redirect' does not exist
    
    ----------------------------------------

```

**Comment by eino-makitalo** [comment]
In windows 10 / virtual env python 3.6 64 bit...
Same...

**Comment by gpshead** [comment]
It looks like I forgot to add `include python3_redirect/*` to `MANIFEST.in`.

Testing... A `pip install .` did not expose this which is likely why I missed it before.
But a `python3 setup.py sdist && pip install dist/subprocess32-3.5.0.rc1.tar.gz` did.
Moral: Always test an install using the actual package distribution file, not the directory tree.

**Comment by mr-c** [comment]
:-D Feel free to adapt my stress-test script: https://github.com/common-workflow-language/cwltool/blob/master/release-test.sh

**Comment by gpshead** [comment]
Fixed in 970fee6764084612acda5e871bf57cc2b8495cbe

**Comment by mr-c** [comment]
Thanks!
